test/OpenTelemetry.Instrumentation.AWS.Tests/TestAWSClientInstrumentation.cs

```diff
@@ -220,9 +217,6 @@ private void ValidateDynamoActivityTags(Activity ddb_activity)
     private void ValidateSqsActivityTags(Activity sqs_activity)
     {
         Assert.Equal("SQS.SendMessage", sqs_activity.DisplayName);
-        Assert.Equal("SQS", Utils.GetTagValue(sqs_activity, "aws.service"));
```

Comment:
Instead of removing these checks for `aws.service` and `aws.operation` shouldn't they updated to check `rpc.service` and `rpc.method`?

Reply:
Yep, they do exist few lines below
![image](https://github.com/user-attachments/assets/a5d31edd-848c-4c51-bd5f-3b7143c9ec53)